pkg/collectors/pool.go

```diff
@@ -127,6 +136,131 @@ func (f *PerfCollector) initPoolDescs() {
 	}
 }
 
+func isPoolInternalMDisks(poolName string, disksList rest.MDisksList) (bool, error) {
+	var internalPool bool
+	for _, disk := range disksList {
+		if poolName == disk[MdiskGroupNameKey].(string) {
+			if disk[ControllerNameKey].(string) != "" {
+				internalPool = false
+			} else {
+				internalPool = true
+			}
```

Comment:
I am not sure when a pool is considered an internal pool (maybe you mean internal storage?), but it seems like we are returning `true` or `false` according to the last disk in the pool - is that what we wanted?

Reply:
fixed